src/Sulu/Bundle/AdminBundle/Admin/View/ListViewBuilderTrait.php

```diff
@@ -62,6 +62,11 @@ private function setSearchableToView(View $route, bool $searchable): void
         $route->setOption('searchable', $searchable);
     }
 
+    private function setSelectableToView(View $route, bool $searchable): void
```

Comment:
The second parameter should probably be `$selectable`

Reply:
👍